content/classes/next-steps.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Next Steps"
+date: 2023-01-23T13:36:52-06:00
+draft: false
+weight: 4
+originalAuthor: <no value> # to be set by page creator
+originalAuthorGitHub: <no value> # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+## Next Steps
+
```

Comment:
Some additional sources I've found helpful:

https://www.tutlane.com/tutorial/csharp/csharp-classes-and-objects-with-examples

Reply:
Added!